<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="regression test"
           tests="5"
           failures="5"
           errors="0"
           skipped="0"
           time="43.836">
  <properties />
            <testcase name="verifyUserShouldAddProductToTheCartSuccessfully" time="6.299" classname="com.automation.testsuits.WomenCategoryPageTest">
                <failure type="java.lang.AssertionError"
                  message="displayed message is wrong  expected [Product successfully added to your shopping cart] but found []"
              >
<![CDATA[java.lang.AssertionError: displayed message is wrong  expected [Product successfully added to your shopping cart] but found []
  org.testng.Assert.fail(Assert.java:110)
  org.testng.Assert.failNotEquals(Assert.java:1413)
  org.testng.Assert.assertEqualsImpl(Assert.java:149)
  org.testng.Assert.assertEquals(Assert.java:131)
  org.testng.Assert.assertEquals(Assert.java:655)
  com.automation.testsuits.WomenCategoryPageTest.verifyUserShouldAddProductToTheCartSuccessfully(WomenCategoryPageTest.java:43)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
  org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
  org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
  org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
  org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
  org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
  org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
  org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
  java.util.ArrayList.forEach(ArrayList.java:1249)
  org.testng.TestRunner.privateRun(TestRunner.java:808)
  org.testng.TestRunner.run(TestRunner.java:603)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
  org.testng.SuiteRunner.run(SuiteRunner.java:326)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
  org.testng.TestNG.runSuites(TestNG.java:1092)
  org.testng.TestNG.run(TestNG.java:1060)
  com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
  com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
      </failure>

          </testcase>
            <testcase name="verifyUserShouldNavigateToWomenCategoryPageSuccessfully" time="1.423" classname="com.automation.testsuits.WomenCategoryPageTest">
                <failure type="java.lang.AssertionError"
                  message="women text is not match expected [WOMWN] but found [WOMEN]"
              >
<![CDATA[java.lang.AssertionError: women text is not match expected [WOMWN] but found [WOMEN]
  org.testng.Assert.fail(Assert.java:110)
  org.testng.Assert.failNotEquals(Assert.java:1413)
  org.testng.Assert.assertEqualsImpl(Assert.java:149)
  org.testng.Assert.assertEquals(Assert.java:131)
  org.testng.Assert.assertEquals(Assert.java:655)
  com.automation.testsuits.WomenCategoryPageTest.verifyUserShouldNavigateToWomenCategoryPageSuccessfully(WomenCategoryPageTest.java:29)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
  org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
  org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
  org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
  org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
  org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
  org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
  org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
  java.util.ArrayList.forEach(ArrayList.java:1249)
  org.testng.TestRunner.privateRun(TestRunner.java:808)
  org.testng.TestRunner.run(TestRunner.java:603)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
  org.testng.SuiteRunner.run(SuiteRunner.java:326)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
  org.testng.TestNG.runSuites(TestNG.java:1092)
  org.testng.TestNG.run(TestNG.java:1060)
  com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
  com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
      </failure>

          </testcase>
            <testcase name="verifyUserShouldAddProductToTheCartSuccessfully" time="6.316" classname="com.automation.testsuits.WomenCategoryPageTest">
                <failure type="java.lang.AssertionError"
                  message="displayed message is wrong  expected [Product successfully added to your shopping cart] but found []"
              >
<![CDATA[java.lang.AssertionError: displayed message is wrong  expected [Product successfully added to your shopping cart] but found []
  org.testng.Assert.fail(Assert.java:110)
  org.testng.Assert.failNotEquals(Assert.java:1413)
  org.testng.Assert.assertEqualsImpl(Assert.java:149)
  org.testng.Assert.assertEquals(Assert.java:131)
  org.testng.Assert.assertEquals(Assert.java:655)
  com.automation.testsuits.WomenCategoryPageTest.verifyUserShouldAddProductToTheCartSuccessfully(WomenCategoryPageTest.java:43)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
  org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
  org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
  org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
  org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
  org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
  org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
  org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
  java.util.ArrayList.forEach(ArrayList.java:1249)
  org.testng.TestRunner.privateRun(TestRunner.java:808)
  org.testng.TestRunner.run(TestRunner.java:603)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
  org.testng.SuiteRunner.run(SuiteRunner.java:326)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
  org.testng.TestNG.runSuites(TestNG.java:1092)
  org.testng.TestNG.run(TestNG.java:1060)
  com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
  com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
      </failure>

          </testcase>
            <testcase name="verifyUserShouldAddProductToTheCartSuccessfully" time="23.558" classname="com.automation.testsuits.WomenCategoryPageTest">
                <failure type="org.openqa.selenium.NoSuchElementException"
                  message="no such element: Unable to locate element: {&quot;method&quot;:&quot;xpath&quot;,&quot;selector&quot;:&quot;//input[@id=&apos;quantity_wanted&apos;]&quot;}
  (Session info: chrome=101.0.4951.54)
For documentation on this error, please visit: https://selenium.dev/exceptions/#no_such_element
Build info: version: &apos;4.1.4&apos;, revision: &apos;535d840ee2&apos;
System info: host: &apos;LAPTOP-N67ITPLC&apos;, ip: &apos;192.168.1.67&apos;, os.name: &apos;Windows 10&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;1.8.0_121&apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [d062acf1f531e3516ac32b58729afbb2, findElement {using=xpath, value=//input[@id=&apos;quantity_wanted&apos;]}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 101.0.4951.54, chrome: {chromedriverVersion: 101.0.4951.41 (93c720db8323..., userDataDir: C:\Users\megha\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:64318}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:64318/devtoo..., se:cdpVersion: 101.0.4951.54, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: d062acf1f531e3516ac32b58729afbb2"
              >
<![CDATA[org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//input[@id='quantity_wanted']"}
  (Session info: chrome=101.0.4951.54)
For documentation on this error, please visit: https://selenium.dev/exceptions/#no_such_element
Build info: version: '4.1.4', revision: '535d840ee2'
System info: host: 'LAPTOP-N67ITPLC', ip: '192.168.1.67', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_121'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [d062acf1f531e3516ac32b58729afbb2, findElement {using=xpath, value=//input[@id='quantity_wanted']}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 101.0.4951.54, chrome: {chromedriverVersion: 101.0.4951.41 (93c720db8323..., userDataDir: C:\Users\megha\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:64318}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:64318/devtoo..., se:cdpVersion: 101.0.4951.54, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: d062acf1f531e3516ac32b58729afbb2
  sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
  sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
  sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
  java.lang.reflect.Constructor.newInstance(Constructor.java:423)
  org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
  org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
  org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:53)
  org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:184)
  org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:167)
  org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:142)
  org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:567)
  org.openqa.selenium.remote.ElementLocation$ElementFinder$2.findElement(ElementLocation.java:162)
  org.openqa.selenium.remote.ElementLocation.findElement(ElementLocation.java:60)
  org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:385)
  org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:377)
  org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:70)
  org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:39)
  com.sun.proxy.$Proxy14.clear(Unknown Source)
  com.automation.pages.ProductPage.enterQuantity(ProductPage.java:30)
  com.automation.testsuits.WomenCategoryPageTest.verifyUserShouldAddProductToTheCartSuccessfully(WomenCategoryPageTest.java:37)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
  org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
  org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
  org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
  org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
  org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
  org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
  org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
  java.util.ArrayList.forEach(ArrayList.java:1249)
  org.testng.TestRunner.privateRun(TestRunner.java:808)
  org.testng.TestRunner.run(TestRunner.java:603)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
  org.testng.SuiteRunner.run(SuiteRunner.java:326)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
  org.testng.TestNG.runSuites(TestNG.java:1092)
  org.testng.TestNG.run(TestNG.java:1060)
  com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
  com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
      </failure>

          </testcase>
            <testcase name="verifyUserShouldAddProductToTheCartSuccessfully" time="6.240" classname="com.automation.testsuits.WomenCategoryPageTest">
                <failure type="java.lang.AssertionError"
                  message="displayed message is wrong  expected [Product successfully added to your shopping cart] but found []"
              >
<![CDATA[java.lang.AssertionError: displayed message is wrong  expected [Product successfully added to your shopping cart] but found []
  org.testng.Assert.fail(Assert.java:110)
  org.testng.Assert.failNotEquals(Assert.java:1413)
  org.testng.Assert.assertEqualsImpl(Assert.java:149)
  org.testng.Assert.assertEquals(Assert.java:131)
  org.testng.Assert.assertEquals(Assert.java:655)
  com.automation.testsuits.WomenCategoryPageTest.verifyUserShouldAddProductToTheCartSuccessfully(WomenCategoryPageTest.java:43)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
  org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
  org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
  org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
  org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
  org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
  org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
  org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
  java.util.ArrayList.forEach(ArrayList.java:1249)
  org.testng.TestRunner.privateRun(TestRunner.java:808)
  org.testng.TestRunner.run(TestRunner.java:603)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
  org.testng.SuiteRunner.run(SuiteRunner.java:326)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
  org.testng.TestNG.runSuites(TestNG.java:1092)
  org.testng.TestNG.run(TestNG.java:1060)
  com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
  com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
